build-apisix-runtime.sh

```diff
@@ -91,8 +91,8 @@ fi
 if [ "$repo" == ngx_multi_upstream_module ]; then
     cp -r "$prev_workdir" ./ngx_multi_upstream_module-${ngx_multi_upstream_module_ver}
 else
-    git clone --depth=1 -b $ngx_multi_upstream_module_ver \
-        https://github.com/api7/ngx_multi_upstream_module.git \
+    git clone --depth=1 -b upgrade_openresty-1.25.3 \
+        https://github.com/zll600/ngx_multi_upstream_module.git \
```

Comment:
please revert this

Reply:
Should we need to add a new tag for `ngx_multi_upstream_module` first?